.changeset/eight-things-attack.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/cli': minor
+---
+
+**BREAKING**: `jest` is now a peer dependency. If you run tests using Backstage CLI, you must add Jest and its environment dependencies as `devDependencies` in your project.
+
+You can choose to install either Jest 29 or Jest 30:
```

Comment:
```suggestion
You can choose to install either Jest 29 or Jest 30. The built-in Jest version before this change was Jest 29, however, we recommend that you switch to Jest 30. Upgrading will solve the `Could not parse CSS stylesheet` errors and ensure that you remain compatible with future versions of the Backstage CLI. Support for Jest 29 is temporary, with the purpose of allowing you to upgrade at your own pace, but it will eventually be removed.
```

I think it's worth nudging users more clearly towards Jest 30